windows/cloud-provider.psm1

```diff
@@ -0,0 +1,139 @@
+$ErrorActionPreference = 'Stop'
+$WarningPreference = 'SilentlyContinue'
+$VerbosePreference = 'SilentlyContinue'
+$DebugPreference = 'SilentlyContinue'
+$InformationPreference = 'SilentlyContinue'
+
+function Log-Info
```

Comment:
seems like logs' function is in every powershell scripts. Can we move them into one file?

Reply:
Refactored, going to test again.